### ISSUE TITLE
AnemoneIcons.dylib causing springboard crash

### ISSUE THREAD

**Issue by shadowscott** [AnemoneTeam/Anemone#203]
iPhone SE upscaled to 6 resolution
iOS 10.2
Only circle icons installed, AnemoneIcons.dylib is causing crash
 
 
Crash log: http://pastebin.com/9Z7Z2EPT
Syslog: http://pastebin.com/aZN42wgC
Tweak list: http://pastebin.com/78CLWHXv

@coolstar if you need anything else please PLEASE lemme know

**Comment by RickyX3** [comment]
I saw that you are using betterfiveicondock, have you used Harbor? You might like it better. Just a suggestion. 😃

**Comment by imrobbyrc** [comment]
same using harbor will crash anemone when applying theme, i was try that,

**Comment by RickyX3** [comment]
Try enabling normal dock background in Harbor

**Comment by coolstar** [comment]
@shadowscott please give a list of your enabled themes

**Comment by shadowscott** [comment]
@coolstar circleicons, bottom option. Courtesy of you
http://i.imgur.com/kYSGgiF.png

**Comment by RickyX3** [comment]
@shadowscott do you have bytafont & have you tried changing themes perhaps that'll fix the crashing.

**Comment by shadowscott** [comment]
@rickyx3 no, my tweak list is above. And I had these crashes even without any themes active. 

@coolstar any luck with the logs ?

**Comment by RickyX3** [comment]
@shadowscott is that your most recent crash log?

**Comment by RickyX3** [comment]
@shadowscott I would try disabling anemone icon effects if you haven't already to see if that might fix it, but I also recommend trying a different theme
